src/vs/workbench/contrib/codeEditor/browser/accessibility/accessibility.ts

```diff
@@ -22,11 +22,15 @@ class ToggleScreenReaderMode extends Action2 {
 			id: 'editor.action.toggleScreenReaderAccessibilityMode',
 			title: { value: nls.localize('toggleScreenReaderMode', "Toggle Screen Reader Accessibility Mode"), original: 'Toggle Screen Reader Accessibility Mode' },
 			f1: true,
-			keybinding: {
+			keybinding: [{
 				primary: KeyMod.CtrlCmd | KeyCode.KeyE,
 				weight: KeybindingWeight.WorkbenchContrib + 10,
 				when: accessibilityHelpIsShown
-			}
+			},
+			{
+				primary: KeyMod.CtrlCmd | KeyCode.F1 | KeyMod.Shift,
+				weight: KeybindingWeight.WorkbenchContrib + 10,
+			}]
```

Comment:
ctrl+shift+f1 is peek implementations so this is no good.

Reply:
Maybe alt+f3, following alt+f1/f2 pattern for a11y keybindings?

Reply:
yeah just about everything is taken that I considered, so sounds good